src/js/utils/ServiceUtil.js

```diff
@@ -28,6 +28,27 @@ const ServiceUtil = {
         definition.instances = formModel.General.instances;
       }
 
+      if (formModel.Optional != null) {
+        definition.executor = formModel.Optional.executor;
+        definition.fetch = formModel.Optional.uris &&
+          formModel.Optional.uris.split(',')
```

Comment:
The `SchemaForm` leverages the key as the title of the section. 
Either way, may I suggest to `let {Optional} = formModel;`? So that we don't have to repeat ourselves so much.